### ISSUE TITLE
Review any changes to model input features/ sources

### ISSUE THREAD

**Issue by torbjorgen** [Clinical-Genomics/rdds#380]
See definition here https://github.com/Clinical-Genomics/rdds/blob/master/src/rdds/variant_rank_score/model/model.py#L65-L90 in Mario release to make sure model is compatible.

Compare MIP (training data set), raredisease v2 (mivmirvalidation modules, 2nd blind validation set) to raredisease v3 Mario.

**Comment by torbjorgen** [comment]
Inspecting the `-profile test` output annotations yields this list of missing annotations:

```raw
- `CSQ/PolyPhen`
- `CSQ/SIFT`
- `CSQ/CLINVAR_CLNREVSTAT` Missing in v3
- `CSQ/CLINVAR_CLNSIG` missing in v3
- `CSQ/MaxEntScan_alt` Missing in v3
- `CSQ/MaxEntScan_diff`
- `CSQ/MES-SWA_acceptor_alt` Missing in v3
- `CSQ/MES-SWA_donor_alt` Missing in v3
- `CSQ/MES-SWA_donor_diff` Missing in v3
- `CSQ/SpliceAI_pred_DS_AL`
- `CSQ/SpliceAI_pred_DS_DG`
- `CSQ/SpliceAI_pred_DS_DL`
- `CSQ/REVEL_score` Missing in v3
- `CSQ/LoFtool`
- `CSQ/GERP++_RS` Missing in v3
- `CSQ/phastCons100way_vertebrate` Missing in v3
- `CSQ/phyloP100way_vertebrate` missing in v3
- `CSQ/SpliceAI_pred_DS_AG`
- `most_severe_consequence`
- `CADD`
- `SWEGENAF` ([SweGen Variant Frequency Dataset](https://swefreq.nbis.se/dataset/SweGen)) missing in v3
- `GNOMADAF_popmax`
- `Frq` (variant frequency from a local database) missing in v3
```

**Comment by torbjorgen** [comment]
Inspecting case `amusingsatyr` run on `raredisease:v3.0.0` I've compiled this list, OK means present:

```raw
CSQ_PolyPhen OK
CSQ_SIFT OK
CSQ_CLINVAR_CLNREVSTAT OK
CSQ_CLINVAR_CLNSIG OK
most_severe_consequence OK
CSQ_MaxEntScan_alt OK
CSQ_MaxEntScan_diff OK
CSQ_MES-SWA_acceptor_alt OK
CSQ_MES-SWA_donor_alt OK
CSQ_MES-SWA_donor_diff OK
CSQ_SpliceAI_pred_DS_AL OK
CSQ_SpliceAI_pred_DS_DG OK
CSQ_SpliceAI_pred_DS_DL OK
CSQ_SpliceAI_pred_DS_AG OK
CSQ_REVEL_score OK
CSQ_LoFtool OK
CSQ_GERP++_RS OK
CSQ_phastCons100way_vertebrate OK
CSQ_phyloP100way_vertebrate OK
CADD OK
SWEGENAF OK

GNOMADAF_popmax MISSING

```

**Comment by torbjorgen** [comment]
Seems like we need to add GNOMADAF_popmax to the annotations in raredisease v3 @ramprasadn ?

**Comment by torbjorgen** [comment]
According to @jemten `GNOMADAF_popmax` is nowadays called `GNOMADAF_grpmax` @ramprasadn ?

**Comment by torbjorgen** [comment]
AFAIK only the GNOMADAF_popmax > grpmax is changed.

Closing this after discussion with @jemten and @ramprasadn
